cmd/docs/main.go

```diff
@@ -0,0 +1,47 @@
+package docs
+
+import (
+	"github.com/datreeio/datree/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+type DocsCommandContext struct {
+	BrowserCtx utils.OpenBrowserContext
+	URL        string
+}
+
+func (do *DocsCommandContext) OpenURL(url string) error {
+	err := do.BrowserCtx.OpenBrowser(url)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func New(ctx *DocsCommandContext) *cobra.Command {
+	var docsCmd = &cobra.Command{
+		Use:     "docs",
+		Short:   "Open datree documentation page",
+		Long:    "Open the default browser with datree documentation page",
+		Aliases: []string{"documentation"},
+		Example: (`
```

Comment:
can you please use `utils.Example` like used in `publish/main.go`?

Reply:
@TzlilSwimmer123 I missed that one, thanks!
@imrushi I think we can merge after you fix this one 🚀

Reply:
@myishay Please check I have made the change.